backend/src/app.js

```diff
@@ -0,0 +1,36 @@
+const fastify = require("fastify")({ logger: true })
+
+const { setupPgBossQueue } = require("./pg")
+const { TWITTER_USER_OBJECT } = require("./twitterUserObjectScraping")
+
+async function buildFastify() {
+  const pgBossQueue = await setupPgBossQueue()
+
+  fastify.route({
+    method: "POST",
+    url: "/twitterUserObject",
```

Comment:
Do we still need this endpoint? I thought we dropped this

Reply:
I use it for worker debugging. I will describe this use case when I extend the docs after I get something working in Heroku.